examples/java-basic-mvn/src/main/java/pineconeexamples/MinimalUpsertAndQueryExample.java

```diff
@@ -53,6 +53,7 @@ public static void main(String[] cliArgs) {
                     .addAllValues(Floats.asList(5F, 3F, 1F))
                     .build();
 
+            // Deprecated: use addValue() or addAllValues() instead of addVector() and addAllVectors() respectively
```

Comment:
Would it make sense to just update to `addValue()` or `addAllValues()` in the example itself rather than having the deprecation comments?

Reply:
I already have examples to show how to use addValue() and addAllValues(). These examples are old and I'll eventually like to get rid of them.